source/assets/mock/form-case-ekb.js

```diff
@@ -24,7 +24,7 @@ const formEkbMockData = {
           footer: {},
           text: {
             colors: {
-              primary: '#00213f',
+              primary: '#00213F',
```

Comment:
should be lowercase

Reply:
Why lowercase? We are handling all other hexcodes with uppercase?

Reply:
just do as I tell you